src/EdFi.AnalyticsMiddleTier.Tests/TestCaseHelper.cs

```diff
@@ -87,6 +98,7 @@ private string GetResultXml()
                 ? $"The expected data set does not match the one received from the database.{Environment.NewLine}Xml Diff Result:{Environment.NewLine}{(result.Root?.ToString() ?? string.Empty)}"
                 : string.Empty;
             return (bIdentical, comparisonResult);
+#pragma warning restore S3265
```

Comment:
Another case where I tried to suppress this error and it is still coming through. I recommend that the pragma continue to be in the source code, so that dotnet build _does_ ignore it at the command line, and then we will have to _also_ dismiss the alert with status "won't fix".